Tests/SwiftSyntaxBuilderTest/ProtocolDeclTests.swift

```diff
@@ -0,0 +1,29 @@
+import XCTest
+import SwiftSyntax
+import SwiftSyntaxBuilder
+
+final class ProtocolDeclTests: XCTestCase {
+  func testProtocolDecl() {
+    let returnClause = ArrayType(elementType: "DeclSyntax")
+    let input = ParameterClause(parameterListBuilder: {
+      FunctionParameter(firstName: .identifier("format"), colon: .colon, type: "Format", trailingComma: .comma, attributesBuilder: {})
+      FunctionParameter(firstName: .identifier("leadingTrivia"), colon: .colon, type: OptionalType(wrappedType: "Trivia"), attributesBuilder: {})
+    })
+    let functionSignature = FunctionSignature(input: input, output: returnClause)
+
+    // FIXME: We need to add the `modifiersBuilder` with a none empty value, otherwise will the builder omit newline.
```

Comment:
Is this comment still up-to-date? I don’t see a `modifiersBuilder` with an empty value.

Reply:
Maybe my comment is not clear, it should not be empty.
`none empty value`

Reply:
Ah, I see. In that case the comment the comment should say `non-empty` instead of `none empty`.